src/ui/components/Events/ReplayInfo.tsx

```diff
@@ -51,12 +51,20 @@ function ReplayInfo({ setModal }: PropsFromRedux) {
           <MaterialIcon iconSize="xl">{icon}</MaterialIcon>
           <div>{summary}</div>
         </Row>
-        <Row>
-          <MaterialIcon iconSize="xl">language</MaterialIcon>
-          <div className="overflow-hidden overflow-ellipsis whitespace-pre" title={recording.url}>
-            {getDisplayedUrl(recording.url)}
-          </div>
-        </Row>
+
+        <div className="group">
+          <Row>
```

Comment:
Suggestion: maybe add a `className` prop to `Row` components, and just pass `group` value to it instead of creating an additional `div.group`?